### ISSUE TITLE
Source code?

### ISSUE THREAD

**Issue by pombredanne** [ossillate-inc/packj#67]
Hi:
I would love to test this tool (I happen to have a few commits in strace) but I cannot trust this non-open source binary plug at
https://github.com/ossillate-inc/packj/blame/4797939a09e5fb113d60577515044e905afd4fa0/packj/sandbox/README.md#L142

You wrote:

> We are not ready to immediately open source this small piece yet. This is NOT to implement security by obscurity, but to avoid easy copy-and-reuse scenarios. The rest of the tool is completely open; this piece will be too, eventually.

When will you make this open source?

**Comment by ashishbijlani** [comment]
We will try to do this soon, but may not happen before the end of the year. However, you could easily write your own strace-based checks and disable loading of the shared library.

**Comment by pombredanne** [comment]
Fair enough! though would you mind to reopen and keep this open until it is fixed?

**Comment by ashishbijlani** [comment]
Sure. Just reopened this.
